README.md

```diff
@@ -67,3 +67,14 @@ Tests can then be run with the following command
 
 ```bash
 poetry run pytest e2e_tests
+```
+
+## Vagrant
+
+To run the app file in vagrant run:
+```bash
+vagrant up
+```
+
+then navigate to http://localhost:5000
+
```

Comment:
Good job remembering to keep your README up to date.
Small correction unrelated to this PR: it should have "Doing" in the list of necessary Trello lists

Also unrelated to anything:
If you require a particular structure but without any files committed inside it (like a secrets directory), you should put an empty file in that folder (by convention called .gitkeep)

But that's made redundant by:
Why do you have both a secrets file and env vars? (Taking the api key and token from environment would be preferable)